source/code/plugins/out_oms_blob.rb

```diff
@@ -184,12 +184,16 @@ def append_blob(uri, msgs, file_path, blocks_committed)
       end
 
       # append blocks
-      # if the msg is longer than 4MB (to be safe, we use 4,000,000), we should break it into multiple blocks
-      chunk_size = 4000000
+      # if the msg is longer than 100MB (to be safe, blob limitation is 100MB), we should break it into multiple blocks
+      chunk_size = 100000000
       blocks_uncommitted = []
-      while msg.to_s.length > 0 do
-        chunk = msg.slice!(0, chunk_size)
```

Comment:
msg.slice!() will still do the slice even if msg size equal or less the chunk_size.
I just run a benchmark with this code which was slicing 100MB from 100MB. and took 44.8 ms.
for slicing 10MB it would be 4ms, It better to save 4ms with a simple if/else.
```
size = 100_000_000
msg = "x" * size
msg.slice!(0, size)
```